### PR TITLE
docs: Add instructions to build & test SCOS

### PR DESCRIPTION
**Pull request by travier**
Initial set of docs for SCOS with current workarounds.

I'll also update the RHCOS docs to match.

### CODE REVIEW

**Review by cgwalters — APPROVED** [review]
Looks sane to me!

I'm hopeful we can land the coreos-assembler bits to streamline this, which would help deduplicate the docs here.

File: docs/development-scos.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+- Clone the config repo (`openshift/os`):
+  ```
+  $ cosa init https://github.com/openshift/os.git
+  ```
+- **Temporary workaround until we have variant support in COSA:** Manually
+  select the SCOS variant:
+  ```
+  $ ln -snf "manifest-c9s.yaml" "src/config/manifest.yaml"
+  $ ln -snf "extensions-c9s.yaml" "src/config/extensions.yaml"
+  ```
```
Comment (bgilbert) [inline docs/development-scos.md:32]: Also need to repoint the `image.yaml` symlink.

File: docs/development-scos.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+- Fetch packages and build SCOS ostree container and QEMU image:
+  ```
+  $ cosa fetch
+  $ cosa build
+  ```
+
+## Building CentOS Stream CoreOS images for other platforms than QEMU
+
+- You can build images for platforms that are supported in COSA using the
+  [`builextend` commands][builextend]:
```
Comment (bgilbert) [inline docs/development-scos.md:54]:
```suggestion
  [`buildextend` commands][buildextend]:
```

File: docs/development-scos.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+- You may then run an ephemeral virtual machine using QEMU with:
+  ```
+  $ cosa run
+  ```
+
+## Testing CentOS Stream CoreOS with kola
+
+- You may then run tests on the image built with [`kola`][kola]:
+  ```
+  $ cosa kola run
```
Comment (bgilbert) [inline docs/development-scos.md:71]: Without `--upgrades` or `--basic-qemu-scenarios`, `cosa kola` is just an alias for `kola` these days, right?  Maybe we should drop the `cosa` part here.

Comment (travier) [reply docs/development-scos.md:71]: We still need the `cosa` part for the users of the cosa container. I'll extend this part.

**Pull request merged**